website/content/docs/concepts/mount-migration.mdx

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+page_title: Mount Migration
+description: Move secrets and auth mounts within and across namespaces
+---
+
+# Mount Migration
+
+Vault supports moving secrets and auth mounts, both within and across namespaces. The operation
+is supported by API endpoints as well as a CLI helper around them.
+
+## Why
+
+There can be several reasons for wanting to migrate mounts. On OSS, the use cases could be for 
+renaming mounts in order to align with org standards. 
+On Enterprise, there are a few more potential use cases as namespaces come into the picture. 
+When migrating from an OSS binary to an Enterprise binary, organizations may want to divide their 
+mounts across several namespaces. Mounts may also be moved across namespaces when there is a reorganization
+of roles and responsibilities.
+
+## Async Behavior
+
+The first thing to note about the `sys/remount` endpoint is that it is an asynchronous endpoint. An invocation
+will start the migration process, and the API will return a migration ID. That ID can in turn be used to monitor
+the status of the migration using the `sys/remount/status` endpoint.
+
+## Namespaces
+
+The `sys/remount` endpoint can move mounts across namespaces with the namespaces supplied as prefixes
+on the path arguments. In this case, namespaces for both the source and destination mounts need to be 
+either the namespace in which the `/sys/remount` endpoint is called, or the children of that namespace. Let us assume that 
+we have 3 namespaces ns1/, ns1/ns2/ and ns1/ns3. We want to move the mount `my-secret` from ns1/ns2/my-secret to ns1/ns3/my-secret. 
+This can be done in one of two ways. The first way is to invoke `/sys/remount` from the root namespace with the `from` argument as ns1/ns2/my-secret 
+and `to` argument as ns1/ns3/my-secret. The second way is to invoke `/sys/remount` from the namespace ns1/, with `from` as ns2/my-secret and `to` 
+as ns3/my-secret.
+
+## Leases
+
+The remount operation will result in existing leases on a mount being revoked. This is potentially the 
+most time consuming part of the operation. Any dynamic secrets for a secrets mount, and any tokens associated
+with an auth mount, will be revoked during the remount process.
+
+## Configurations
+
+The remount operation will retain all the configurations of the mount. For example, if you create a readonly role
+on a database mount, or a user on an userpass mount, after remounting those will be available at the new location.
+
+## Cleanup operations
+
+There are 2 main cleanup operations that Vault operators will need to keep in mind after performing a remount.
+
+### Entities and aliases
+
+When an auth mount is moved, any aliases that refer to users on the auth mount could now point to an invalid mount.
```

Comment:
```suggestion
Any aliases that refer to users on the auth mount could now point to an invalid mount when an auth mount is removed. Pointing to an invalid mount may not be the case for every instance;
```

Reply:
think this should be `moved` or `remounted` not `removed` ?

Reply:
```suggestion
Any aliases that refer to users on the auth mount could now point to an invalid mount when an auth mount is moved. Pointing to an invalid mount may not be the case for every instance;
```